python/ray/data/_internal/execution/interfaces/physical_operator.py

```diff
@@ -169,6 +170,15 @@ def on_task_finished(self):
         self._task_done_callback()
 
 
+@dataclass
+class _ActorInfo:
+    """Breakdown of the state of the actors used by the ``PhysicalOperator``"""
+
+    running: int
+    pending: int
+    restarting: int
+
```

Comment:
Nit: I felt kinda confused by these name. `_ActorInfo` makes me think a single actor rather than all of the actors for an operator, and `running` etc. seemed like bools rather than counts. 

Maybe something like `ActorStateCounts` or `OperatorActorStats` and `num_running` etc. might be clearer

Reply:
+1

Reply:
I was calling it OpActorInfo initially, but it's now used inside the ActorPool. Will rename it to _ActorPoolInfo